src/sqlancer/questdb/test/QuestDBQueryPartitioningBase.java

```diff
@@ -67,6 +67,11 @@ protected ExpressionGenerator<Node<QuestDBExpression>> getGen() {
     public void check() throws SQLException {
         s = state.getSchema();
         targetTables = s.getRandomTableNonEmptyTables();
+        if (targetTables.getTables().size() > 1) {
```

Comment:
I think it would be clearer to add a method `s.getRandomTable()` that directly returns a random table.